.github/workflows/push-to-registry.yml

```
@@ -31,7 +31,7 @@ jobs:
       id: login-ecr
       uses: aws-actions/amazon-ecr-login@v1
 
-    - name: Build and Push to ECR public
+    - name: Push docking-autodock4 to ECR
```

```suggestion
    - name: Build and Push to ECR public
```
Step name should either go back to what it was, or be "docking-smina". Autodock4 step is still "Build and Push to ECR public" so we should keep it consistent

---

yeah, something got mixed up in a merge here